pkg/cstor/volume/v1alpha1/build.go

```diff
@@ -203,7 +204,17 @@ func (b *Builder) WithCapacity(capacity string) *Builder {
 		)
 		return b
 	}
-	b.cstorvolume.object.Spec.Capacity = capacity
+	capacityQnt, err := resource.ParseQuantity(capacity)
+	if err != nil {
+		b.errs = append(
+			b.errs,
+			errors.Wrapf(
+				err, "failed to update capacity: failed to parse capacity {%s}",
+				capacity,
```

Comment:
error message is verbose but is it helpful.

Is this same as we follow everywhere?
What operation failed is not known?
Updating capacity may be done at various places. It does not talk about specific operation.

Reply:
updated error message